README.md

```diff
@@ -57,12 +57,38 @@ This project supports a basic [Vagrant](http://docs.vagrantup.com/v2/getting-sta
 
 1. Run vagrant up command
 
-    vagrant up
+        vagrant up
```

Comment:
@akrabat Code blocks boundary is the common way used in most documents, also allow to include language information.